src/mod_muc.erl

```diff
@@ -561,7 +542,7 @@ route_presence_to_nonexistent_room(Room, From, To, Acc, Packet,
             ejabberd_router:route(To, From, Acc, Err)
```

Comment:
Lack of proper return value after this line, that could be used by `route_to_room`.